RealmSwift/Util.swift

```diff
@@ -27,6 +27,15 @@ internal func noWarnUnsafeBitCast<T, U>(_ x: T, to type: U.Type) -> U {
     return unsafeBitCast(x, to: type)
 }
 
+internal func unwrap(arguments: [Any]) -> [Any] {
```

Comment:
This seems like an awfully generic name for such a special-purpose function. Not sure how much that matters, though, given that it's for internal use only.